app/[locale]/academics/convocation/page.tsx

```diff
@@ -1,9 +1,286 @@
+        glyphDirection="rtl"
+        heading="h3"
+        id="about"
+        text={text.about}
+      />
+      <article
+        id="about"
+        className="container flex flex-row-reverse drop-shadow max-md:flex-col"
+      >
+        <Image
+          alt={'About'}
+          className="w-full object-cover max-md:rounded-b md:order-first md:rounded-l"
+          height={0}
+          width={0}
+          src={`https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg`}
+        />
+        <p
+          className={cn(
+            'p-2 sm:p-3 md:p-4',
+            'bg-neutral-50 max-md:rounded-t md:w-full md:rounded-r'
+          )}
+        >
+          Graduation promises the donning of a new and challenging life. The end
+          of college life marks the beginning of a journey that our mentors and
+          parents had been preparing us for. The fun and frolic, the ups and
+          downs, are all small stepping stones to the stage where one has to
+          honestly play the role of an able and responsible citizen. NIT
+          Kurukshetra witnessed its 19th Convocation with a lots of overwhelmed
+          students. The college has conferred the various degrees to 1000+
+          students till now and brimming with enthusiasm and preparations NIT
+          Kurukshetra is set to reward students with various degrees,
+          scholarships, and merits of hard work. Indian president Smt. Droupadi
+          Murmu had graced the occasion with her benign presence.
+        </p>
+      </article>
+
+      {/* Dgree recipent */}
+      <Heading
+        className="container"
+        glyphDirection="ltr"
+        heading="h3"
+        id="degree-recipent"
+        text={text.guest}
+      />
+      <article
+        id="Cheef-Guest-Message"
+        className="container flex drop-shadow max-md:flex-col"
+      >
+        <p
+          className={cn(
+            'p-2 sm:p-3 md:p-4',
+            'bg-neutral-50 max-md:rounded-t md:w-full md:rounded-r'
+          )}
+        >
+          Indian youth, my friends at NIT Kurukshetra! This day celebrates your
+          remarkable accomplishments and the potential you hold. The education
+          you&apos;ve received here is a powerful weapon. Wield it not just for
+          personal gain, but to build a better India. Innovate, challenge
+          yourselves, and never shy away from solving problems. You are the
+          architects of our nation&apos;s tomorrow. Make it a future brimming
+          with progress, a testament to the brilliance nurtured within these
+          very walls. Congratulations, and best wishes on your journeys ahead!
+        </p>
+        <Image
+          alt={'About'}
+          className="w-full object-cover max-md:rounded-b md:order-first md:rounded-l"
+          height={0}
+          width={0}
+          src={`https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg`}
+        />
+      </article>
```

Comment:
This doesn't exist in the current website. We are only mapping what already exists to the new website; we cannot create our own data.